perma_web/frontend/components/UploadForm.vue

```diff
@@ -137,8 +141,13 @@ defineExpose({
             "This will create a new Perma Link." }}
             </p>
             <div class="modal-body">
+
+                <div class="u-min-h-48" v-if="isLoading">
+                    <Spinner length="2" color="#2D76EE" top="48px" />
+                </div>
```

Comment:
The loading spinner (which still relies on legacy spinner.js code) doesn't typically take up space where it is used elsewhere on the dashboard. Adding a div, with a utility class here with a min-height, felt like the easiest way to add it for this specific situation.